activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -132,12 +132,16 @@ def initialize(*addresses)
       # Increment a cached value. This method uses the memcached incr atomic
       # operator and can only be used on values written with the +:raw+ option.
       # Calling it on a value not stored with +:raw+ will initialize that value
-      # to zero.
+      # to zero. :initial option can be used for initializing the value to 
```

Comment:
> Calling it on a value not stored with +:raw+ will initialize that value to zero.

This seems to be describing an old version of the Memcached text protocol:

https://github.com/memcached/memcached/blob/1.6.15/doc/protocol.txt#L359-L362

> If the current data value does not conform to such a representation, the incr/decr commands return an error (memcached <= 1.2.6 treated the bogus value as if it were 0, leading to confusion).

Since Rails 4.0 MemCacheStore uses Dalli, which uses the newer binary protocol, so this comment is no longer correct; we should fix it while we're here. I like the wording from https://github.com/rails/rails/pull/40581 ("fail and return nil").